thunderfish/lizpulses.py

```diff
@@ -57,43 +63,61 @@ def extract_pulsefish(data, samplerate, peakwidth=0.002, cutwidth=0.001, **clust
         For each detected fish, the unreliability score, where scores > 0.1
         signify unreliable EOD means (possibly wavefish or unreliable clusters).
     """
-    
+
     mean_eods, eod_times, eod_unreliability = [], [], []
     
     # extract peaks
+    if verbose>0:

```

Comment:
In general, your verbose info looks good. But I have two comments on how to improve it:

1. The information what is going to happen next is not needed, unless some information about the input is printed out. The verbose comments are mainly meant for the advanced user that needs to tweak some parameters and therefore needs to see how it changes the outcome of whatever information. So, only verbose infos if they spit out some info that depends on the data. The particular 'Extracting EOD times' (and similar) can be removed.

2. Put the verbosity information into the function that is generating this information. The particular 'Extracting EOD times' (if not removed because of 1.) thus should go into the extract_eod_times() function. And this function is then called like this:
```
extract_eod_times( ... verbose-1, ....)
```